### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by eduardoconceicao**
## Description
Update dependencies:
* Update `eslint` to `v6.5.1`
* Replace `eslint-config-seegno` by `eslint-config-uphold`
* Add `prettier@1.18.2`
*

### CODE REVIEW

**Comment by waldyrious** [comment]
@eduardoconceicao you can create draft PRs instead of using the `wip` label :)

File: .eslintrc.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-extends: seegno
+extends: uphold
 env:
   jasmine: true
   jest: true
```
Comment (waldyrious) [inline .eslintrc.yml]: This entire file seems [redundant](https://github.com/uphold/eslint-config-uphold/blob/v1.0.1/src/index.js#L6-L12).

Comment (eduardoconceicao) [reply .eslintrc.yml]: Nice catch @waldyrious ;)

File: package.json [hunk 2]
```diff
@@ -20,16 +20,20 @@
...
+    "jest": "^24.9.0",
     "koa": "^2.0.1",
     "lint-staged": "^3.4.0",
     "pre-commit": "^1.2.2",
+    "prettier": "^1.18.2",
     "supertest": "^3.0.0"
   },
   "engines": {
-    "node": ">=7.6"
+    "node": "^8.10.0 || ^10.13.0 || >=11.10.1"
```
Comment (rplopes) [inline package.json]: Simplify? The rules for eslint shouldn't have to matter to projects that depend on this one.

File: package.json [hunk 3]
```diff
@@ -20,16 +20,20 @@
...
+  },
+  "eslintConfig": {
```
Comment (rplopes) [inline package.json]: This seems to be against our conventions. Keep in `.eslintrc.yml`?

Comment (waldyrious) [reply package.json]: Good point. It's best to have a single way to do things than to change depending on context.

**Pull request merged**